app/views/report/_form_sort.html.haml

```diff
@@ -148,7 +148,7 @@
                     options_for_select(MiqReport::GROUPINGS.map(&:reverse).map { |key, value| [_(key), value] }, selected),
                     :class    => "selectpicker show-tick",
                     :multiple => true,
-                    "title"   => "Check Option")
+                    "title"   => _("Check Option"))
```

Comment:
The string is marked for translation but what else do I need to do to translate it?

Reply:
cc @mzazrivec

Reply:
From a devel point of view, the work here is done. To see the string translated you'd need new catalogs with the translations.